shared/repository/lib/api/documents/ContactRepository.mjs

```diff
@@ -18,37 +19,29 @@ export class UserPhonesRepository extends RepoBase {
 
     /**
      * @method
-     * @param {number} userId - user identifier
-     * @param {boolean} tracked - the flag that indicates do the phone is tracking
+     * @param {number} id - User's contact idetifier
+     * @param {number} userId - User identifier
+     * @param {boolean} tracked - The flag that indicates is the contact tracked
+     * @param {string} trackedPhone - Contact added by user that sholud be tracked
+     * @param {string} createdAt - Entry creation date
+     * @param {string} updatedAt - Entry update date
+     * @param {string} SessionId - Idetifier of session that tracks this contact
      * @return {Promise<Object>} returns data saved in DB
      */
-    async read({ userId, tracked }) {
-        let sql = 'SELECT * FROM tracked_phones WHERE user_id = $1';
-        const params = [userId];
-        if (typeof tracked === 'boolean') {
-            sql += ' AND tracked = $2';
-            params.push(tracked);
-        }
-        const result = await this.db.queryAsync(sql, params).catch((err) => {
-            throw new RepoError(err);
-        });
-        return result && result.rows;
-    }
 
-        /**
-     * @method
-     * @param {number} userId - user identifier
-     * @param {string} phone - users phone is tracking
-     * @return {Promise<Object>} returns data saved in DB
-     */
-         async readOne({ userId, phone }) {
-            const sql = 'SELECT * FROM tracked_phones WHERE user_id = $1 AND tracked_phone = $2';
-            const params = [userId, phone];
-            const result = await this.db.queryAsync(sql, params).catch((err) => {
-                throw new RepoError(err);
-            });
-            return result && result.rows[0];
-        }
+    async read({ id, userId, trackedPhone, tracked, createdAt, updatedAt, sessionId }) {
+        const dbProps = {
+            id,
+            user_id: userId,
+            tracked_phone: trackedPhone,
+            tracked,
+            created_at: createdAt,
+            updated_at: updatedAt,
+            session_id: sessionId,
+        };
+
+        return await dbRead(this.db, 'tracked_phones', dbProps);
```

Comment:
do not use a very common method for selecting from DB. it is hard to debug and read